src/modules/m_sqloper.cpp

```diff
@@ -24,9 +24,9 @@
 class OpMeQuery : public SQLQuery
 {
  public:
-	const std::string uid, username, password;
-	OpMeQuery(Module* me, const std::string& u, const std::string& un, const std::string& pw)
-		: SQLQuery(me), uid(u), username(un), password(pw)
+	const std::string uid, username, password, hashtype;
```

Comment:
I did not understand why hashtype should be a const ref and not uid, username or password.